From: Kefeng Wang <wangkefeng.wang@huawei.com>
Subject: mm: hugetlb: remove left over comment about follow_huge_foo()
Date: Thu, 25 Jul 2024 10:16:43 +0800

The comment is useless after commit 57a196a58421 ("hugetlb: simplify
hugetlb handling in follow_page_mask") since all follow_huge_foo() are
killed.

Link: https://lkml.kernel.org/r/20240725021643.1358536-1-wangkefeng.wang@huawei.com
Signed-off-by: Kefeng Wang <wangkefeng.wang@huawei.com>
Reviewed-by: David Hildenbrand <david@redhat.com>
Reviewed-by: Vishal Moola (Oracle) <vishal.moola@gmail.com>
Cc: Muchun Song <muchun.song@linux.dev>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/hugetlb.c |    4 ----
 1 file changed, 4 deletions(-)

--- a/mm/hugetlb.c~mm-hugetlb-remove-left-over-comment-about-follow_huge_foo
+++ a/mm/hugetlb.c
@@ -7503,10 +7503,6 @@ __weak unsigned long hugetlb_mask_last_p
 
 #endif /* CONFIG_ARCH_WANT_GENERAL_HUGETLB */
 
-/*
- * These functions are overwritable if your architecture needs its own
- * behavior.
- */
 bool isolate_hugetlb(struct folio *folio, struct list_head *list)
 {
 	bool ret = true;
_
